core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -568,6 +568,8 @@
   // accumulo garbage collector properties
   GC_PREFIX("gc.", null, PropertyType.PREFIX,
       "Properties in this category affect the behavior of the accumulo garbage collector."),
+  GC_CANDIDATE_BATCH_SIZE("gc.candidate.batch.size", "4000000", PropertyType.COUNT,
+          "The batch size used for garbage collection. The default is effectively 8MB"),
```

Comment:
If we make this property of type `PropertyType.BYTES`, then it may be more user friendly, and the comment about the default can be removed, because it would be redundant. If you do that, then it's a simple matter to divide by 2 to get the approximate number of characters for the count in the implementation. Using `PropertyType.BYTES` will also make it more user friendly, because users can use the typed-strings, like `16M` or `1G` or `512K` when they change the configuration (and the default can be `8M`).